salt/_modules/metalk8s_volumes.py

```diff
@@ -13,6 +13,8 @@
 
 from salt.exceptions import CommandExecutionError
 
+import metalk8s_blkid as blkid
```

Comment:
Again, can't do this. Need to look things up from `__salt__` or something across modules IIRC.

Reply:
Yup, now using `__utils__`.